src/sdk/bucketspace_fs.js

```diff
@@ -382,6 +383,7 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
                     if (list && list.objects && list.objects.length > 0) {
                         throw new RpcError('NOT_EMPTY', 'underlying directory has files in it');
                     }
+                    await native_fs_utils.folder_delete(namespace_bucket_config.write_resource.path, this.fs_context);
```

Comment:
shouldn't it be the temp folder and not the whole folder?

Reply:
@romayalon S3 bucket delete will delete the complete folder? If the folder is not empty it will not allow delete, right?

Reply:
in general yes, 
but this is the else scope in which we shouldn't delete the whole directory because should_create_underlying_storage=false, here we should only delete the internal directory of the export (by the specific bucket id)

Reply:
make sense, will do the changes